src/node.rs

```diff
@@ -1021,83 +1072,60 @@ impl<S: ForkSource + std::fmt::Debug> InMemoryNode<S> {
+        for call in &call_traces {
             inner.console_log_handler.handle_call_recurive(call);
         }
-
         log::info!("");
         log::info!(
             "==== {} Use --show-calls flag or call config_setShowCalls to display more info.",
-            format!("{:?} call traces. ", tx_result.call_traces.len()).bold()
+            format!("{:?} call traces. ", call_traces.len()).bold()
         );
 
         if inner.show_calls != ShowCalls::None {
-            for call in &tx_result.call_traces {
+            for call in &call_traces {
                 formatter::print_call(call, 0, &inner.show_calls, inner.resolve_hashes);
             }
         }
-
         log::info!("");
         log::info!(
             "==== {}",
-            format!("{} events", tx_result.result.logs.events.len()).bold()
+            format!("{} events", tx_result.logs.events.len()).bold()
         );
-        for event in &tx_result.result.logs.events {
+        for event in &tx_result.logs.events {
             formatter::print_event(event, inner.resolve_hashes);
         }
 
-        // Compute gas details
-        let debug = BootloaderDebug::load_from_memory(&vm).map_err(|err| err.to_string())?;
-
-        // Total amount of gas (should match tx.gas_limit).
-        let gas_limit = debug
-            .total_gas_limit_from_user
-            .saturating_sub(debug.reserved_gas);
-
-        let intrinsic_gas = gas_limit - debug.gas_limit_after_intrinsic;
-        let gas_for_validation = debug.gas_limit_after_intrinsic - debug.gas_after_validation;
-
-        let gas_spent_on_compute =
-            debug.gas_spent_on_execution - debug.gas_spent_on_bytecode_preparation;
-
-        let gas_used = intrinsic_gas
-            + gas_for_validation
-            + debug.gas_spent_on_bytecode_preparation
-            + gas_spent_on_compute;
-
         // The computed block hash here will be different than that in production.
-        let hash = compute_hash(block_context.block_number, l2_tx.hash());
+        let hash = compute_hash(batch_env.number.0, l2_tx.hash());
         let block = Block {
             hash,
             number: U64::from(inner.current_miniblock.saturating_add(1)),
-            timestamp: U256::from(block_context.block_timestamp),
-            l1_batch_number: Some(U64::from(block_context.block_number)),
+            timestamp: U256::from(batch_env.timestamp),
+            l1_batch_number: Some(U64::from(batch_env.number.0)),
             transactions: vec![TransactionVariant::Full(
                 zksync_types::api::Transaction::from(l2_tx.clone()),
             )],
-            gas_used,
-            gas_limit,
+            gas_used: U256::from(tx_result.statistics.gas_used),
+            gas_limit: U256::from(BLOCK_GAS_LIMIT),
             ..Default::default()
         };
 
-        log::info!("");
-        log::info!("");
-
-        vm.execute_till_block_end(BootloaderJobType::BlockPostprocessing);
+        log::info!("\n\n");
```

Comment:
Please revert for consistency of logs
```suggestion
        log::info!("");
        log::info!("");
```